internal/backends/gl/Cargo.toml

```diff
@@ -62,13 +66,11 @@ glutin = { version = "0.28", default-features = false }
 usvg = { version= "0.20", optional = true, default-features = false, features = ["text", "memmap-fonts"] }
 
 [target.'cfg(target_family = "windows")'.dependencies]
-font-kit = { version = "0.10", features = [] }
+#font-kit = { version = "0.10", features = [] }
 
 [target.'cfg(not(any(target_family = "windows", target_os = "macos", target_os = "ios", target_arch = "wasm32")))'.dependencies]
 libc = { version = "0.2" }
-# Require font-config from the system on Linux. Issue #88 indicates that the copy provided by servo-fontconfig may be incompatible
-# with distros at times.
-servo-fontconfig = { version = "0.5", features = [ "force_system_lib" ] }
+yeslogic-fontconfig-sys = { git = "https://github.com/Be-ing/fontconfig-rs.git", branch = "dlopen" }
```

Comment:
I think for us this switch would be alright (from servo-fontconfig to yeslogic's fontconfig). I hope the servo/fontkit folks will agree to switching from servo-fontconfig to yeslogic.